### DISCUSSION TITLE
Did wofi start acting weird reccently for anyone else?

### DISCUSSION THREAD

**Discussion by stephanlamoureux** [hyprwm/Hyprland#6611] [Q&A]
Wofi has lost its ability to blur, I also get a ghosting effect as I scroll through the apps, and the background color disappears around the edges.  It launches correctly for a second:

![before](https://github.com/hyprwm/Hyprland/assets/16889593/27c71e39-cd2b-4d83-ac98-87fd52458cc9)

and then turns into this: 

![20240620_16h19m59s_grim](https://github.com/hyprwm/Hyprland/assets/16889593/f9d03b53-2c22-4811-92ab-80dbd73d3f4d)

**Comment by abenezerw** [comment]
Did you happen to solve this @stephanlamoureux
